src/inc/conpty-static.h

```diff
@@ -15,6 +15,8 @@
 extern "C" {
 #endif
 
+#define PSEUDOCONSOLE_RESIZE_QUIRK (2u)

```

Comment:
If this is a flag, it's definition should be of the form `0x10`, not `2u`, to symbolize flag-ness.